src/pages/Login.vue

```diff
@@ -7,15 +7,15 @@
 
                 <h1>Log In</h1>
 
+                <div class="message">The username and password combination entered was not recognised. Please check your details and try again.</div>
```

Comment:
Fin to start but we probably need these to be the messages returned by the API. Are there any there?